ratbagd/ratbagd-device.c

```diff
@@ -137,6 +137,29 @@ static int ratbagd_device_get_device_name(sd_bus *bus,
 	return 0;
 }
 
+static int ratbagd_device_get_device_type(sd_bus *bus,
+					  const char *path,
+					  const char *interface,
+					  const char *property,
+					  sd_bus_message *reply,
+					  void *userdata,
+					  sd_bus_error *error)
+{
+	struct ratbagd_device *device = userdata;
+	uint32_t devicetype;
+
+	devicetype = ratbag_device_get_device_type(device->lib_device);
+	if (!devicetype) {
+		log_error("%s: device type unspecified\n",
+			  ratbagd_device_get_sysname(device));
+		devicetype = 0;
```

Comment:
shouldn't be here `devicetype = TYPE_UNSPECIFIED`? After all, that's why one defines enums in the first place.

Reply:
I agree. currently the enum is inside `libratbag-data.c` which means it cannot be accessed outside of that file.

Can I just put it inside of `libratbag.h`? Then it would be accessable in all the spots you suggested.

Reply:
Maybe put it inside `src/libratbag-enums.h` and include that if necessary.

Reply:
oh that makes sense, i did not see that file. Will do.